master/reference/kube-controllers/configuration.md

```diff
@@ -80,6 +81,12 @@ To enable the node controller, perform the following two steps.
 
 This controller is only valid when using etcd as the {{site.prodname}} datastore.
 
+Additionally, the node controller can be configure to keeplabels on Kubernetes Node 
+objects and Calico Node objects in sync. By setting `SYNC_NODE_LABELS` to true (enabled 
```

Comment:
```
s/Calico Node/{{site.nodecontainer}}`
```

Reply:
Same here - this isn't referring to `calico/node` (But, I think "Node" should be lowercase)

Reply:
yup, but i did lower case the node :)